gast_to_code.py

```diff
@@ -1,45 +1,51 @@
-
-# general_helpers
-def value_to_str(val):
-    if type(val) == str:
-        return '"' + val + '"'
-    else:
-        return str(val)
-
-def body(gast_list, out_lang):
+"""
+Helper for lists of gast
+Default is to put comma and space btwn each stringified gast
+    i.e. list_helper({str_gast}, {str_gast}, out_lang) --> str, str
+Can specify different btwn string with third parameter
+    i.e. list_helper({str_gast}, {str_gast}, out_lang, "**") --> str**str
+"""
+def list_helper(gast_list, out_lang, btwn_str = ", "):
     out = ""
     for gast in gast_list:
         out += gast_router(gast, out_lang)
-        out += "\n"
-
-    return out[:-1] # remove \nS
-
-def list_to_csv_str(l):
-    s = ""
-    for i in l:
-        if type(i) == str:
-            i = '"' + i + '"'
-        s += str(i) + ", "
+        out += btwn_str
     
-    return s[:-2] # remove last comma and space
-
+    num_btwn_chars = len(btwn_str)
```

Comment:
`csv_delimiter_len = ...`

Reply:
Better would be to just `break` from the loop early instead of removing the string after the fact. You can grab the index in a for in loop using `enumerate()` and check if it's the last element. This logic is a little awkward as is.

https://stackoverflow.com/a/522578

Reply:
done!